src/App.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import './App.css';
+import Home from './components/Home';
+import Details from './components/Details';
+import Header from './components/Header';
+import NotFound from './components/NotFound';
+
+function App() {
+  return (
+    <div className="App">
+      <BrowserRouter>
+        <Header />
+        <Routes>
+          <Route path="/" element={<Home />} />
+          <Route path="details/:location" element={<Details />}>
```

Comment:
- [ ] Each item on the home page should have a unique route. Right now, all your items have the same route: `/details`, but they should be something like so: `/details/:id`. Kindly fix that.

  For example (you don't need to use the same word), instead of all your routes having the same route like so:

  ```
  /details
  ```
  They can have this:
  ```
  /details/chipper-hill

  /details/san-antonio
  ```

Reply:
Thank you for the review and feedback @TedLivist 
I will make sure to make the requested changes and submit fo another review. 🙏